src/dfx/src/lib/environment.rs

```diff
@@ -284,12 +295,20 @@ impl<'a> AgentEnvironment<'a> {
                 and use it in mainnet-facing commands with the `--identity` flag", identity.name());
         }
         let url = network_descriptor.first_provider()?;
+        let effective_canister_id = network_descriptor
+            .local_server_descriptor
+            .as_ref()
+            .and_then(|d| {
+                d.is_pocketic()
+                    .then(|| Principal::from_text("tqzl2-p7777-77776-aaaaa-cai").unwrap())
```

Comment:
Please follow the method used by MAINNET_LEDGER_CANISTER_ID, MAINNET_CYCLE_MINTER_CANISTER_ID, and others, for defining const Principals